apps/aecore/lib/aecore/structures/transaction.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Aecore.Structures.Transaction do
```

Comment:
does this function as interface?

Reply:
Yes, it's functionality is as to an interface